### PR TITLE
feat: added CI build test

### LINKED ISSUE #1871: CI Automated Test for Docker image Build
### Problem

Sometimes when upgrading the node version things can break due to unintended changes on nodejs.

Like in this PR:
https://github.com/hashgraph/hedera-json-rpc-relay/pull/1844/files

### Solution

Create an automated CI Test that performs the following actions:

1. Build the Docker image (with the new node version to upgrade)
2. Use the newly created docker image for starting the relay as part of the hedera-local-node.
3. Run the acceptance tests against that image

This CI Test should run on demand, whenever we update the node version.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
added CI build test

**Related issue(s)**:

Fixes #1871 

**Notes for reviewer**:
when I did one of the offending changes commits it did fail as expected:

**failed build:**
https://github.com/hashgraph/hedera-json-rpc-relay/actions/runs/8973110170/job/24642544534?pr=2456

image

**after reverting the offending commit:**
image

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/image-build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          driver-opts: network=host
+
+      - name: Build image
+        uses: docker/build-push-action@v2
+        with:
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          platforms: linux/amd64, linux/arm64
+          push: false
```
Comment (Nana-EC) [inline .github/workflows/image-build.yml:32]: Q: Is there anything we can do to test the image or no?

**Review by Nana-EC — APPROVED** [review]
LG.
let's see if there's a valuable next step that is an easy check on the built image

File: .github/workflows/image-build.yml (hunk 1 above)
Comment (AlfredoG87) [reply .github/workflows/image-build.yml:32]: Yes, just added steps for that...

image

**Review by Nana-EC — APPROVED** [review]

**Comment by AlfredoG87** [comment]
> LG. let's see if there's a valuable next step that is an easy check on the built image

I did the `/health/liveness` on the built image, are u thinking in something else? like actual RPC requests?

**Pull request merged**